reana-ui/src/util.js

```diff
@@ -125,6 +125,15 @@ export function parseFiles(files) {
   return files;
 }
 
+/**
+ * Parses search input term.
+ * @param {String} term Search term
+ * @returns term format expected by the API.
+ */
+export function parseSearch(term) {
```

Comment:
Minor question about the naming. Does this function really `parses` search input term? As I understand it produces a query param, so this function call could be misleading to someone. Perhaps `getSearchParam` or something like that would be more meaningful? WDYT?

Reply:
Yeah, I wasn't very sure either, I followed other function naming but the logic is not really the same. `getSearchParam` is a bit better but I'm still not convinced, maybe `encodeSearch`?

Reply:
One of the hardest problems in CS..
`encodeSearch` sounds nice to me! Though it might be mixed with encoding of strings. Another solution would be simply `formatSearch`?

Reply:
Fair enough 👍